# Request
int32 sample_num
string log_filename
---
# Results
bool success
float32[] base
---
# Feedback
bool f
